tests/daemon/test-daemon.c

```diff
@@ -1225,48 +1175,13 @@ test_daemon_does_not_record_sequences_when_daemon_disabled (Fixture      *fixtur
 }
 
 static void
-test_daemon_updates_timestamps_on_shutdown (Fixture      *fixture,
```

Comment:
Not sure I follow. Are you saying the test should be added with the same `finalize` modification as the flushing test?